src/library/trackset/crate/cratefeature.cpp

```diff
@@ -28,7 +28,7 @@ namespace {
 
 QString formatLabel(
         const CrateSummary& crateSummary) {
-    return QString("%1 (%2) %3")
+    return QStringLiteral("%1 (%2) %3")
             .arg(
```

Comment:
Does QStringLiteral actually work with `.arg()`? I remember I tried that once and it didn't work...

Reply:
I can confirm that it does work and also remember that it didn't previously. Maybe switching to QStringBuilder did the trick?

Reply:
The tests are all succeeding, and I hope there is at least _some_ testing covering the library. I changed this in so many places that it would be shocking if it wouldn't work and the tests would still succeed.